features/collection/shell-commands/src/main/java/org/opennms/netmgt/collection/commands/CollectCommand.java

```diff
@@ -122,12 +141,32 @@ public Void execute() {
                 .withAttributes(parse(attributes))
                 .execute();
 
+        Persister persister = null;
+        if (persist) {
+            ServiceParameters params = new ServiceParameters(Collections.emptyMap());
+            RrdRepository repository = new RrdRepository();
+            persister = persisterFactory.createPersister(params, repository);
+            if (rras != null && rras.size() > 0) {
+                repository.setRraList(rras);
+            } else {
+                repository.setRraList(Lists.newArrayList(
```

Comment:
Can we defined this list as a constant and share it with https://github.com/OpenNMS/opennms/blob/cdc8610caa2708ca5437daef2e537b0eaf409933/features/collection/shell-commands/src/main/java/org/opennms/netmgt/collection/commands/StressCommand.java#L201 instead of duplicate it here?